ui_src/src/domain/overview/tags/index.js

```diff
@@ -27,25 +27,27 @@ const Tags = () => {
                     <div className="overview-components-header">
                         <p>Most used tags</p>
                     </div>
-                    {state?.monitor_data?.tags_details?.map((tag, index) => {
-                        return (
-                            <div key={index}>
-                                <span className="tag-item">
-                                    <span className="item">
-                                        <label className="item-num">{`${index + 1}`}</label>
-                                        <Tag tag={{ color: tag.color, name: tag.name }} onClick={() => ''}></Tag>
+                    <div className="tags-items-container">
+                        {state?.monitor_data?.tags_details?.map((tag, index) => {
+                            return (
+                                <div key={index}>
+                                    <span className="tag-item">
+                                        <span className="item">
+                                            <label className="item-num">{`${index + 1}`}</label>
+                                            <Tag tag={{ color: tag.color, name: tag.name }} onClick={() => ''}></Tag>
```

Comment:
Don't pass onClick and handle it inside the Tag component.
Also, tags shouldn't be clickable.

Reply:
done